src/main/web/templates/handlebars/partials/header.handlebars

```diff
@@ -17,7 +17,7 @@
 							<img class="logo" src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.png" alt="Office for National Statistics">
 						<![endif]-->
 						<!--[if gte IE 9]><!-->
-							<img class="logo" src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg" alt="Office for National Statistics">
+							<img class="logo" src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg" alt="Office for National Statistics logo - homepage">
```

Comment:
Is homepage needed?

Reply:
I believe so; as per DAC report, which suggested including 'homepage' in the `alt` so as to make it clearer that the image links back to the homepage

Reply:
I missed that good point